lib/src/datePicker/safe-html.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { DomSanitizer } from "@angular/platform-browser";
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({ name: "safeHtml" })
+export class SafeHtmlPipe implements PipeTransform {
```

Comment:
If not mistaken, there is similar pipe, I wonder if we should create a folder to store the pipes/ functions that can be reused by each component

Reply:
yeah I moved it to __utils folder and others can reuse it instead of creating their own (By others I mean @DanSallau hahaha)